package.json

```diff
@@ -70,7 +70,7 @@
     "gulp-sourcemaps": "^1.2.4",
     "gulp-uglify": "^1.0.1",
     "gulp-webserver": "^0.9.1",
-    "jest-cli": "0.4.19",
+    "jest-cli": "0.6.0",
```

Comment:
We have `jest-cli` at `0.4.19` because later versions require `iojs` or Node 5, and making the necessary changes for those is out of scope of this PR. I'll have to undo this change before merging.